sample/app/src/main/proguard-rules.pro

```diff
@@ -0,0 +1,2 @@
+-keep class org.jetbrains.annotations.** { *; }
+-dontwarn org.jetbrains.annotations.**
```

Comment:
This file is not needed can be removed, I've added it to test pro guard for bazel

Reply:
deleted